### ISSUE TITLE
null - when I run balena login on windows 10 balena-cli

### ISSUE THREAD

**Issue by lpt2007** [balena-io/balena-cli#1331]
I installed balena-cli from [here](https://github.com/balena-io/balena-cli/releases/download/v11.4.1/balena-cli-v11.4.1-windows-x64-installer-BETA.exe) and when I run balena login from command prompt I get null. Why?

<img width="305" alt="balena_null" src="https://user-images.githubusercontent.com/38924279/60398700-a1c6cb80-9b5b-11e9-80a2-f16f9f382c6c.png">

**Comment by lpt2007** [comment]
I figured out. Problem is in installer verrsion. [Standalone](https://github.com/balena-io/balena-cli/releases/download/v11.4.1/balena-cli-v11.4.1-windows-x64-standalone.zip) is working.

**Comment by pdcastro** [comment]
Thank you for reporting this issue. I understand you have closed this issue because you have found a workaround by using the standalone installer, but I was able to reproduce the error and I am working on a fix. Another workaround would be to use Windows executable installer for version 11.4.0. I have created two further issues, as they have both combined to result in the "null" error reported here:
* [CLI prints "null" and exits with error code 1](https://github.com/balena-io/balena-cli/issues/1333)
* [Error: Cannot find module 'web-streams-polyfill'](https://github.com/balena-io/balena-cli/issues/1332)
